examples/react-agent-structured-output.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# How to create a function calling agent with structured output\n",
+    "# Force function calling agent to structure output\n",
```

Comment:
sorry for this one let's actualladd "How to" (that's how we've been doing it for other how-tos as well, so "How to" in the notebook title, and no how to in the TOC